proposals/issue_triage.md

```diff
@@ -49,6 +55,21 @@ The following are necessary and sufficient conditions for an issue to be conside
 
   * The oncall will attempt to satisfy the above criterion or reassign to an appropriate WG if there is some question
 
+## Become a contributor
+
+* Make sure that you have enough permissions to assign labels to an issue and add it to a project.
+* In order to get permissions, open a PR to add yourself to [project-maintainers](https://github.com/kubeflow/internal-acls/blob/4e44f623ea4df32132b2e8a973ed0f0dce4f4139/github-orgs/kubeflow/org.yaml#L389) group.
+
+## Triage guideline
+
+* Take an issue from "Needs Triage" project and open it in a new tab.
+* Carefully read the description.
+* Carefully read all comments below. (Some issues might be already resolved).
+* Make sure that issue is still relevant. (Some issues might be open for months and still be relevant to current Kubeflow release whereas some might be outdated and can be closed).
+* Ping one of the issue repliers if he/she is not replying for a while.
+* Make sure that all triage conditions are satisfied.
+* Remove issue from "Needs Triage" Project.
```

Comment:
You can delete this last step about manually removing the issue. 
The issue will be removed the next time we run the script to identity issues needing triage.
If the issue isn't fully triaged it will just be readed.

Ideally we'd run that script  as a GitHub so the addition/removal from the triage board would happen in real time as issues are created/removed.